CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Exported `App-Settings` for cli and other apps to use [#840](https://github.com/zowe/imperative/issues/840)
```

Comment:
Could you please change `App-Settings` to `AppSettings` to match the name of the interface? Sorry for being picky 😢

Reply:
okay